OpenEdXMobile/res/values/strings.xml

```diff
@@ -176,7 +176,7 @@
     <!-- Row of announcement on dashboard screen -->
     <string name="announcement_title">Announcements</string>
     <!-- Title text for the important dates section on the course dashboard  -->
-    <string name="course_dates_title">Important Dates</string>
+    <string name="course_dates_title" translatable="false">Dates</string>
```

Comment:
We should be using `tools:ignore="MissingTranslation"` instead of `translatable="false"`.